tests/Helpers/PsrLoggerTraitTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/*
+ * This file is part of the Bushido\Foundation package.
+ *
+ * (c) Wojciech Nowicki <wnowicki@me.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace BushidoTests\Foundation\Helpers;
+
+use Bushido\Foundation\Helpers\PsrLoggerTrait;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+class PsrLoggerTraitTest extends TestCase
+{
+    use PsrLoggerTrait;
+
+    private $logger;
```

Comment:
Maybe annotate as LoggerInterface? It's internal, so I'm approving it.

Reply:
Done